spec/models/donation_spec.rb

```diff
@@ -7,4 +7,22 @@
   it { is_expected.to validate_presence_of(:donor) }
   it { is_expected.to validate_presence_of(:amount) }
   it { is_expected.to validate_numericality_of(:amount).is_greater_than(0) }
+
+  describe ".search" do
+    let(:smithie_black) { create(:donor, name: "Smithie Black", identification: "G1234567M") }
+    let(:match_beginning) { create(:donation, amount: 100, donor: smithie_black) }
+
+    let(:bob_smith_white) { create(:donor, name: "Bob Smith-White", identification: "G1234567M") }
+    let(:match_middle) { create(:donation, amount: 100, donor: bob_smith_white) }
+
+    let(:john_smith) { create(:donor, name: "John Smith", identification: "G1234567M") }
+    let(:match_end) { create(:donation, amount: 100, donor: john_smith) }
+
+    let(:carl_grey) { create(:donor, name: "Carl Grey", identification: "G1234567M") }
+    let(:no_match) { create(:donation, amount: 100, donor: carl_grey) }
+
+    it do
```

Comment:
Is RuboCop complaining about line length here? If not, let's use `{}` for single line blocks. 😀